[["file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/doctrine/instantiator/tests/DoctrineTest/InstantiatorTestAsset/ExceptionAsset.php",{"_uri":"file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/doctrine/instantiator/tests/DoctrineTest/InstantiatorTestAsset/ExceptionAsset.php","_root":{"kind":4096,"name":"file:///home/jfvilladiego/proyectos_laravel/cotizaciones_vcb_laravel/vendor/doctrine/instantiator/tests/DoctrineTest/InstantiatorTestAsset/ExceptionAsset.php","location":{"uriHash":656161489,"range":{"start":{"line":0,"character":0},"end":{"line":41,"character":0}}},"children":[{"kind":512,"name":"DoctrineTest\\InstantiatorTestAsset","location":{"uriHash":656161489,"range":{"start":{"line":19,"character":0},"end":{"line":19,"character":45}}}},{"kind":1,"name":"BadMethodCallException","location":{"uriHash":656161489,"range":{"start":{"line":21,"character":4},"end":{"line":21,"character":26}}},"modifiers":4096,"associated":[{"kind":1,"name":"BadMethodCallException"}]},{"kind":1,"name":"Exception","location":{"uriHash":656161489,"range":{"start":{"line":22,"character":4},"end":{"line":22,"character":13}}},"modifiers":4096,"associated":[{"kind":1,"name":"Exception"}]},{"kind":1,"name":"DoctrineTest\\InstantiatorTestAsset\\ExceptionAsset","location":{"uriHash":656161489,"range":{"start":{"line":29,"character":0},"end":{"line":40,"character":1}}},"children":[{"kind":32,"name":"__construct","location":{"uriHash":656161489,"range":{"start":{"line":36,"character":4},"end":{"line":39,"character":5}}},"doc":{"description":"Constructor - should not be called","type":""},"modifiers":1,"type":"","children":[],"scope":"DoctrineTest\\InstantiatorTestAsset\\ExceptionAsset"}],"associated":[{"kind":1,"name":"Exception"}],"doc":{"description":"Test asset that extends an internal PHP base exception","type":""},"modifiers":0}]},"_hash":656161489}]]